packages/app/src/elements/import-dialog.ts

```diff
@@ -281,9 +271,17 @@ export class ImportDialog extends Dialog<File, void> {
                 this._items = await imp.asLastPass(file);
                 break;
             case imp.CSV.value:
-                const result = await imp.asCSV(file, this._itemColumns, this._csvHasColumnsOnFirstRowButton.active);
+                const result = await imp.asCSV(
+                    file,
+                    resetCSVColumns ? [] : this._itemColumns,
+                    this._csvHasColumnsOnFirstRowButton.active
+                );
```

Comment:
Item columns need to be reset whenever the `hasColumnsOnFirstRow` value changes, but resetting the value to `[]` before re-parsing causes the UI to be updated twice which leads to some ugly jumpiness.